src/components/fields/contact-field/contact-field.tsx

```diff
@@ -118,9 +120,11 @@ export const ContactField = (props: IGenericFieldProps<IContactFieldSchema>) =>
 			const schemaPrefix = getPrefixFromCountry(countryValue || "Singapore");
 			if (!fixedCountry || (fixedCountry && prefixWithoutPlus === schemaPrefix)) {
 				const fieldPrefix = prefix || schemaPrefix;
-				handleChange({ countryCode: fieldPrefix, number });
+				const phoneNumberWithPrefix = parseAndApplyNumber({ countryCode: fieldPrefix, number });
+				setValue(id, phoneNumberWithPrefix, { shouldDirty: false });
```

Comment:
default `shouldDirty` is false so the second args can be omitted

Reply:
have removed various instances of it